frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -120,6 +150,126 @@ export const SwitchEnvFeedbackModal = ({
                 <FormControl>
                   <Input type="hidden" {...register('url')} value={url} />
                 </FormControl>
+                <FormControl isRequired={true} isInvalid={!isEmpty(errors)}>
```

Comment:
structure seems a little off, this `FormControl` element wraps a single radio group field, but its validity is constrained by all errors. should we update it to only care about the errors it needs?

nit: also true props do not need a `true` assignment

```tsx
<FormControl isRequired isInvalid={!!errors.radio || !!errors[FEEDBACK_OTHERS_INPUT_NAME]}>
  ...
  // As such, only need to render appropriate messages
  <FormErrorMessage>
    {errors.radio?.message ?? errors[FEEDBACK_OTHERS_INPUT_NAME]?.message}
  </FormErrorMessage>
</FormControl>
```

Reply:
https://github.com/opengovsg/FormSG/pull/5245/commits/215307aefcbc89beeab0ae70e456032cb73fc71f

also modified FormErrorMessage bc the error icon for the radio group was showing up even when an unrelated error was triggered (email input) https://github.com/opengovsg/FormSG/pull/5245/commits/641ec3a772d394181457b22f7260128d0f0769d4